src/borg/hashindex.pyx

```diff
@@ -22,7 +22,7 @@ cdef extern from "_hashindex.c":
         uint32_t version
         char hash[16]
 
-    HashIndex *hashindex_read(object file_py, int permit_compact) except *
+    HashIndex *hashindex_read(object file_py, int permit_compact) except NULL
```

Comment:
http://cython.readthedocs.io/en/latest/src/reference/language_basics.html#checking-return-values-for-non-cython-functions

that was the first thing that showed up when i searched for "cython except NULL".